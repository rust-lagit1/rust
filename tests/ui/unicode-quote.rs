fn main() {
    dbg!("since when is \“THIS\” not allowed in a string literal");
}
