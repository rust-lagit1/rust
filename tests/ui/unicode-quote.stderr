error: unknown character escape: `\u{201c}`
  --> $DIR/unicode-quote.rs:2:26
   |
LL |     dbg!("since when is \“THIS\” not allowed in a string literal");
   |                          ^ unknown character escape
   |
   = help: \u{201c} is not an ascii quote, but may look like one in some fonts.
           consider writing it in its escaped form for clarity.

error: unknown character escape: `\u{201d}`
  --> $DIR/unicode-quote.rs:2:32
   |
LL |     dbg!("since when is \“THIS\” not allowed in a string literal");
   |                                ^ unknown character escape
   |
   = help: \u{201d} is not an ascii quote, but may look like one in some fonts.
           consider writing it in its escaped form for clarity.

error: aborting due to 2 previous errors

